modules/pool/pool.service.ts

```diff
@@ -283,6 +284,10 @@ export class PoolService {
             if (networkconfig.data.subgraphs.aura) {
                 await syncAuraStakingForPools(chain, new AuraSubgraphService(networkconfig.data.subgraphs.aura));
             }
+
+            if (networkContext.chain === 'MAINNET') {
+                await syncVebalStakingForPools();
+            }
```

Comment:
i see it could be part of the current flow, but it adds quite a bit of external calls. moving it out makes both parts more resilient. setting up a separate controller + action just for syncing vebal won't need networkContext.